.github/workflows/pull-request-links.yml

```diff
@@ -0,0 +1,16 @@
+name: Add RTD link to PR
+on:
+  pull_request_target:
+    types:
+      - opened
+
+permissions:
+  pull-requests: write
+
+jobs:
+  pull-request-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: readthedocs/actions/preview@v1
+        with:
+          project-slug: "ccn-template"
```

Comment:
I have since removed this, but this action automatically adds a link to the PR build of readthedocs, and is a work-around if you can't give readthedocs the proper access to appear as a status check. flatiron is willing to do so now, so I don't think it's an issue. I had described this at the bottom of docs/notes/03-documentation.md, and there's still a bit there.